python/system/ksp-allow-python-django-idconfig-core.yaml

```diff
@@ -0,0 +1,102 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-allow-python-django-idconfig-core
+  namespace: default
+spec:
+  tags: ["PYTHON", "DJANGO"]
+  message: "Alert! idconfig and core files are only allowed"
```

Comment:
config files access from python bin is allowed only. something like this @yasin-cs-ko-ak